### ISSUE TITLE
Connection to RDP

### ISSUE THREAD

**Issue by mshakurov** [mRemoteNG/mRemoteNG#1492]
## Expected Behavior

Connect to RDP....

## Current Behavior

Nothing but error log...

## Steps to Reproduce (for bugs)

1. Select RDP connection setting
2. Connect
3.
4.

## Your Environment

* Version used: 1.76.20.24615
* Operating System and version (e.g. Windows 10 1709 x64): Microsoft Windows [Version 10.0.17763.529]

Log file:

--- app loading ---

2019-06-17 12:38:05,586 [1] INFO  - mRemoteNG 1.76.20.24615 starting.
2019-06-17 12:38:05,643 [1] INFO  - Command Line: C:\Program Files (x86)\mRemoteNG\mRemoteNG.exe
2019-06-17 12:38:16,795 [1] INFO  - Майкрософт Windows 10 Корпоративная  64-bit
2019-06-17 12:38:16,795 [1] INFO  - Microsoft .NET CLR 4.0.30319.42000
2019-06-17 12:38:16,796 [1] INFO  - System Culture: ru-RU/русский (Россия)
2019-06-17 12:38:16,800 [1] INFO  - Checking FIPS Policy...
2019-06-17 12:38:16,806 [1] INFO  - Checking Lenovo AutoScroll Utility...
2019-06-17 12:38:16,841 [1] DEBUG - Parsing cmdline arguments
2019-06-17 12:38:28,255 [1] INFO  - Override Culture: en/English
2019-06-17 12:38:28,596 [1] INFO  - Loading External Apps from: C:\Users\Marat\AppData\Roaming\mRemoteNG\extApps.xml
2019-06-17 12:38:31,278 [1] DEBUG - Connections loaded using XmlConnectionsLoader
2019-06-17 12:38:33,261 [1] WARN  - PuttySessions.Watcher.StartWatching() failed.
Не найдено 
2019-06-17 12:38:33,262 [1] WARN  - XmingPortablePuttySessions.Watcher.StartWatching() failed: 'sessions' does not exist.
2019-06-17 12:38:33,263 [1] DEBUG - Determining if we need a database syncronizer

--- RDP connecting ---

2019-06-17 12:44:23,847 [1] ERROR - RDP SetProps failed!
Value does not fall within the expected range.
   at MSTSCLib.IMsRdpClient8.set_Server(String pServer)
   at mRemoteNG.Connection.Protocol.RDP.RdpProtocol.Initialize() in C:\jenkins\workspace\Publish\Build mRemoteNG for Release\mRemoteV1\Connection\Protocol\RDP\RdpProtocol.cs:line 129
2019-06-17 12:44:23,874 [4] INFO  - Connection Event Closed
2019-06-17 12:44:23,885 [4] INFO  - Connection to shakurov@st.loc via RDP closed by user Marat.
-------------------------------

**Comment by sparerd** [comment]
@mshakurov I suspect this has to do with using `username@host`. We currently do not support that style of host name parsing. When creating connections, place the username in the `Username` field.

**Comment by ner00** [comment]
This could be added as an enhancement rather than closed. I suppose it wouldn't be very hard to parse the string and split it between the username and domain just by using the `@` as delimiter.

**Comment by farosch** [comment]
@ner00 No offense, but I suppose it wouldn't be very hard to use the appropriate fields for `username` and `password` either, would it?
This just adds complexicity to the code without any benefit and might even break things for some users.
For example. How should we handle passwords, when there are both field set?

> Username = user@password
> Password = password2

Is the password `password` or `password2` ?

And even worse:
> Username = user@user@password

The username could be `user` or `user@user` and the password could be `user@password` or `password`.
I know thats a wierd and rare case but at least linux does allow for the @ character in usernames and we do need to support this.

**Comment by ner00** [comment]
@farosch None taken, sarcasm asside. I was actually just commenting on the wide used format of `user@domain` for Windows logins, but maybe I misunderstood. Apologies.

**Comment by farosch** [comment]
@ner00 No need to apologize - it's always good to get feedback from the community :+1:
Unfortunately in this case implementing user@pass is not so easy as we are supporting various protocols and might need to support other protocols that we don't even know about yet in the future.
Having username and password seperated allows for easier handling of the various protocols.

**Comment by ner00** [comment]
I have the feeling that I'm not getting through, in none of the comments did I suggest using the password in the same string. If I were, I would probably suggest existing formats such as `username: password@hostname`
